tutorials/proxy-protocol-v2-load-balancer/index.mdx

```diff
@@ -117,18 +142,19 @@ The `proxy_protocol` parameter must be set within the `http {}` block of the lis
 
             listen 80   proxy_protocol;
 
-            set_real_ip_from 51.159.26.16;
+            set_real_ip_from {LB_IP};
             real_ip_header proxy_protocol;
 
             #...
           }
+        access_log /var/log/nginx/access.log logs;
       }
```

Comment:
There are two more spaces than needed, closing brace should be aligned with the `h` from `http` on line 131